hack/make/dependencies.make

```diff
@@ -81,25 +85,26 @@ $(HELM):
 
 .PHONY: yq
 yq: $(YQ)
-	test -s $(LOCALBIN)/yq || GOBIN=$(LOCALBIN) go install github.com/mikefarah/yq/v4@$(YQ_VERSION)
+$(YQ):
+	test -s $(YQ) || GOBIN=$(LOCALBIN) go install github.com/mikefarah/yq/v4@$(YQ_VERSION)
```

Comment:
Why remove the `yq` phony target? I assume local-setup is not working now?

Reply:
My mistake, you didn't remove it, but you added this for some reason https://github.com/Kuadrant/multicluster-gateway-controller/pull/320/files#diff-edf16a6750c4ce8ab32c9359e0f357d15f8c79fb01baba58ae1dba663e556b50R89. What's that doing?

Reply:
ignore me, its fine.

Reply:
hmm. i thought I was getting errors without this but running it again it seems it's not necessary. A similar syntax is used in other make commands in this file. Although I'm not sure also why it would be required.

Reply:
What you did is correct, it's using make to check that the file already exists and do nothing if it does (Actually using make the way it was intended to be used). The `test -s $(YQ) ` part is actually redundant now, you could remove it, but i see it also being used on other targets unnecessarily .